cluster/cluster.go

```diff
@@ -9,13 +9,15 @@ import (
 	"bufio"
 	"bytes"
 	"context"
+	joinerrs "errors"
```

Comment:
I'm curious why the "errors" package was renamed from it's default name.

Reply:
We have another package called `errors` that we already use "github.com/pkg/errors". I didn't want to risk switching out the existing uses of that package for the new one and they would have a name conflict otherwise so I just brought in the new one under an alias for the specific use case.

Reply:
On the off chance we use more things from the stdlib `errors` library, maybe we can rename `joinerr` to `stderr` or something similar to reflect it's the Go standard error lib.